### ISSUE TITLE
High memory usage

### ISSUE THREAD

**Issue by navhaxs** [homov/LibMpv#2]
Hey @homov, firstly thanks for putting together this Avalonia wrapper 👍🏼 👍🏼 

I do have a question - when I maximise the window, the memory usage grows without bound. Any ideas why? 
e.g. 10GB after 30 seconds, 20GB after ~1 minute:
![image](https://github.com/homov/LibMpv/assets/5162718/37a1a561-edf5-4bd9-9e32-14aba807d074)

It grows very quickly!

![Snipaste_2023-06-05_22-16-49](https://github.com/homov/LibMpv/assets/5162718/c8a93a1c-53e2-47f1-a6ab-67511e4edbdc)
![Snipaste_2023-06-05_22-17-06](https://github.com/homov/LibMpv/assets/5162718/7425f468-5b47-47e9-9e1c-bebb81918e0c)

**Comment by homov** [comment]
Hello!
I was unable to replicate your result. When I run I don't see this problem.

[Memory consumption when playing in full Window](https://drive.google.com/file/d/1edRsqYsyCbA6F3R2j50sRKS6esQ0rnev/view?usp=sharing)

**Comment by navhaxs** [comment]
Ah I figured out how to reproduce. It only happens when I maximise on my big monitor.

You can add `Height="1440" Width="2560"` to the control, it also triggers the problem this way.

```
<TabItem Header="OpenGL">
	<Viewbox>
		<local:OpenGlVideoView MpvContext="{Binding OpenGLMpvContext}" Height="1440" Width="2560" />
	</Viewbox>
</TabItem>
<TabItem Header="Software">
	<Viewbox>
		<local:SoftwareVideoView MpvContext="{Binding SoftwareMpvContext}" Height="1440" Width="2560"/>
	</Viewbox>
</TabItem>
```

**Comment by homov** [comment]
Yes you are right. When using opengl, I see the same effect. This effect is not present in software rendering. So far I have no idea why this is happening.

**Comment by homov** [comment]
I also tried on Linux. Works without this problem.
Most likely on Windows you need to use native rendering, and on other platforms opengl rendering.
I can assume that this is directly related to libmpv opengl rendering on Windows.

**Comment by homov** [comment]
Apparently, the problem is solved in the latest beta versions of Avalonia. Confirm if you can.

**Comment by navhaxs** [comment]
![image](https://github.com/homov/LibMpv/assets/5162718/558287b5-2533-48c1-916c-92ce5ba87149)

Thanks @homov just tested, looks fixed to me 👍🏼
